README.md

```diff
@@ -0,0 +1,48 @@
+# TEKSI - Wastewater
+
+This repo holds all code related to the TEKSI Wastewater module (formerly QGEP).
+
+It contains:
+- the datamodel definition
+- a QGIS plugin
+- a QGIS project
+- the documentation
+
+
+## Release policy
+
+Teksi-Wastewater follows a *rolling release* model, meaning that the `main` branch always constitutes the latest stable release. **Merging a pull request means deploying a new stable release.**
+
+A yearly release (`YYYY.0`) is created at the same time as QGIS LTR releases (usually in march). Non-backwards compatible changes such as dropping support for dependencies or removing existing workflows can only be merged during a period of 1 month before the yearly release (matching QGIS ).
+
+Two additional tags (`YYYY.1`, `YYYY.2`) are created in august and november. They only serve as milestones for organisational purposes (roadmapping) but otherwise do no differ from rolling release.
+
+To accommodate that release model, the following rules apply:
+- all changes happen in pull requests
+- automated tests must be successfull before merged
+- pull requests must be reviewed and approved by at least 1 other developer
+- pull requests must be manually tested and approved by at least 1 user
+- non-backwards compatible pull requests (dropping support for dependencies, removing existing workflows, etc.) must be tagged with the `frozen` tag
+

```

Comment:
describe `feature thaw`: approx. 1 month before the YYYY.0 release, we enter into `feature thaw` period, during which the version gets increment to `YYYY.0-rc1`, `YYYY.0-rc2`, etc, published as pre-releases.

to be defined how exactly the CI does this (probably the major version name in the source, and CI automatically incrementing the last number, which works both the rc and point releases)